zerver/lib/import_realm.py

```diff
@@ -969,6 +969,9 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
         if "zerver_usergroup" in data:
             update_model_ids(UserGroup, data, "usergroup")
             re_map_foreign_keys(data, "zerver_usergroup", "realm", related_table="realm")
+            re_map_foreign_keys(
+                data, "zerver_usergroup", "can_mention_group", related_table="usergroup"
+            )
```

Comment:
I am not sure whether we need to consider the case where the export data is old like the data contains user group but not `can_mention_group_id` field.

Reply:
We do not -- we only support importing after exporting from the same version.